system/ee/ExpressionEngine/Addons/pages/tab.pages.php

```diff
@@ -118,6 +120,37 @@ public function display($channel_id, $entry_id = '')
         return $settings;
     }
 
+    /**
+     * Clones the page data for cloned entry
+     *
+     * @param ExpressionEngine\Model\Channel\ChannelEntry $entry
+     * @param array $values An associative array of field => value
+     * @return array $values modified array of values
+     */
+    public function clone(ChannelEntry $entry, $values)
+    {
+        if ($values['pages_uri'] == '') {
+            return $values;
+        }
+        //check if submitted URI exists
+        $static_pages = ee()->config->item('site_pages');
+        $uris = $static_pages[ee()->config->item('site_id')]['uris'];
+
+        //exclude current page from check
+        if (isset($uris[$entry->entry_id])) {
+            unset($uris[$entry->entry_id]);
+        }
+        //ensure leading slash is present
+        $value = '/' . trim($values['pages_uri'], '/');
+
+        while (in_array($value, $uris)) {
+            $value .= '_1';
+        }
```

Comment:
I'm thinking we should check to confirm this URL isn't in the array already, and increment the number if it is... if people use the same base entry for cloning, we would then have duplicate urls ending in _1.

Reply:
Parsing the string to get number would be more resource consuming then appending a string, which we do here. We might as well use `_copy` or `_clone` instead of `_1`